source/slang/slang.cpp

```diff
@@ -1965,6 +1974,87 @@ void Module::_addEntryPoint(EntryPoint* entryPoint)
     m_entryPoints.add(entryPoint);
 }
 
+static bool _canExportDeclSymbol(ASTNodeType type)
+{
+    switch (type)
+    {
+        case ASTNodeType::ModuleDecl:
+        case ASTNodeType::EmptyDecl:
+        case ASTNodeType::NamespaceDecl:
```

Comment:
I'm kind of surprised at why `NamespaceDecl` would be on the excluded list here, since a `namespace` declaration isn't very different from a `struct` declaration that just contains `static` members.

Is there a specific reason it had to be left out?